src/backend/drm/surface/gbm.rs

```diff
@@ -23,10 +25,16 @@ pub struct GbmBufferedSurface<A: Allocator<Buffer = BufferObject<()>> + 'static,
     current_fb: Slot<BufferObject<()>>,
     pending_fb: Option<(Slot<BufferObject<()>>, U)>,
     #[allow(clippy::type_complexity)]
-    queued_fb: Option<(Slot<BufferObject<()>>, Option<Vec<Rectangle<i32, Physical>>>, U)>,
+    queued_fb: Option<(
```

Comment:
(bo, sync, damage, user data)?

Reply:
yeah, that starts to get ugly. replaced with an internal struct.